.travis.yml

```diff
@@ -35,3 +38,10 @@ matrix:
       env: 'ESLINT=5 JEST=25'
     - node_js: '8'
```

Comment:
we can run jest 25 on node 8, the bug was fixed. not really for this PR though 🙂

Reply:
Ah I see. I was going to ask :)